core/includes/path.inc

```diff
@@ -55,11 +55,16 @@ function backdrop_lookup_path($action, $path = '', $langcode = NULL) {
     $cache = array(
       'map' => array(),
       'no_source' => array(),
-      'whitelist' => NULL,
+      'allowlist' => NULL,
       'system_paths' => array(),
       'no_aliases' => array(),
       'first_call' => TRUE,
     );
+    // Backwards-compatible removal of `whitelist`. Remove in Backdrop 2.0.
```

Comment:
I could use some guidance as to whether this BC layer is necessary here. Would flushing all caches after the update be sufficient? If so I'd prefer to remove this.

Reply:
I think it's better to just flush the cache once in an update hook and leave this out.